docker-entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# https://elrey.casa/bash/scripting/harden
+set -${-//[sc]/}eu${DEBUG:+xv}o pipefail
```

Comment:
So, this is to help during the runtime of the script, when the script runs if it has any errors it will fail early instead of trying to continue to run. This will help with troubleshooting as well if they are having issue just with even the shell script setting up the environment. You can just pass a `-e DEBUG=true` and it will do 2 things.

1) `-v` - it will print out the whole script ( this will help ensure that someone hasn't modified it and you are running what is expected )
2) `-x` - will show what is getting executed and returned values from commands that are executing. ( i.e. if you are expecting something to return 2, but you see it is getting set to 1 having this flag will tell you this information ).

It is also kind of similar ( but my additions are better for debugging than theirs IMO ) to how the docker library images use with their shell scripts:
https://github.com/docker-library/mysql/blob/c506174eab8ae160f56483e8d72410f8f1e1470f/8.0/docker-entrypoint.sh#L2

I can change it to this if you want,
```suggestion
set -eu${DEBUG:+xv}o pipefail
```
 because the beginning part is mainly for when you are doing a `curl '<url>' | bash`

if you really want me to I can, since this is the communities project ( not my own ( that is what my fork is for :upside_down_face: ) ), but I wanted to explain why I added it :slightly_smiling_face:

Reply:
Thanks! Gotta love apache!

Reply:
simplified with my most recent commit.